source/administration/compliance-export.rst

```diff
@@ -0,0 +1,51 @@
+
+By default, Mattermost stores all message history providing an unlimited search history to admins and end users. In Enterprise Edition E20, you may set a `custom data retention policy <https://docs.mattermost.com/administration/data-retention.html>`_ for how long messages and file uploads are kept in Mattermost channels and direct messages.
+
+Those Enterprise deployments who want to archive history beyond the data retention period can enable this add-on to export compliance reports to third-party systems. Integration with Actiance Vantage is currently supported, with integrations with other systems such as GlobalRelay in the roadmap.
+
+.. note::
+  This feature will replace the existing :doc:`Compliance feature <compliance>` in a future release. Compliance exports to CSV will continue to be available in Enterprise Edition E20.
+
+.. toctree::
+    :maxdepth: 2
+
+Set Up Guide
+--------------
+
+1. Go to **System Console > Advanced > Compliance Export (Beta)**.
+2. Enable compliance exports, then set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.
+3. Set the export file format. Currently, export format to Actiance XML is supported, but support for the GlobalRelay EML format and the Mattermost CSV format is scheduled for a future release.
+4. Set the export directory to write compliance export files to. Must be a subdirectory of the Local Storage Directory. Mattermost must have write access to this directory, and the path that you set must not include a filename.
+5. Save the settings. You’re now all set!
+
+The compliance exports do not contain messages sent before the feature was enabled, but you can export past history via the ``export`` :doc:`command line tool <command-line-tools>`.
+
+Actiance Vantage Integration
+---------------------------------
+
+If you have chosen your file format to be Actiance XML, you can set up an integration with Actiance Vantage.
+
+For more information on Actiance Vantage archive system, see `their homepage <https://www.actiance.com/products/vantage/>`_.
+
+Frequently Asked Questions (FAQ)
+---------------------------------
+
+How do I export past history?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the ``export`` :doc:`command line tool <command-line-tools>`. You can specify an `exportFrom` option to export data from a specified timestamp. All posts that were made after this timestamp will be exported.
+
+What happens if I export data manually?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the compliance export job is run automatically, manually via the System Console, or manually via the CLI (without the ``--exportFrom`` option), it exports all posts that were made since the last message that the previous job exported. If this is the first time that the job has ever run, all posts that were made since the feature was enabled will be exported.
```

Comment:
Should be "it exports all posts that were made since the last post that the previous execution of the job exported." Let's avoid using "message" and "post" as synonyms as much as possible, in an effort to keep our wording consistent

Reply:
Thanks, will update